@nuxt/ui cannot be installed in a Vue 3 + Inertia 3 project because of a peer dependency conflict

### Environment

Vue: 3.5.13
Inertia Vue: 3.0.0
npm: latest
App type: Laravel + Inertia + Vue SPA

### Is this bug related to Nuxt or Vue?

Vue

### Package

v4.x

### Version

v4.6

### Reproduction

When running:
```
npm install @nuxt/ui
```

npm reports that @nuxt/ui@4.6.0 has an optional peer dependency on:
``` 
@inertiajs/vue3@"^2.0.7"
```

but the project already uses:
``` 
@inertiajs/vue3@3.0.0
```

So installation is blocked even though the package is being added to a Vue app, not a Nuxt app.
This appears to affect Vue applications using Inertia 3 even though the app is otherwise compatible with Vue 3.

### Description

I’m trying to install @nuxt/ui into a Laravel + Vue 3 application that uses @inertiajs/vue3@3.0.0, but npm fails with an ERESOLVE dependency conflict.

### Additional context

_No response_

### Logs

```shell-script
npm error ERESOLVE could not resolve
npm error While resolving: @nuxt/ui@4.6.0
npm error Found: @inertiajs/vue3@3.0.0
npm error Could not resolve dependency:
npm error peerOptional @inertiajs/vue3@"^2.0.7" from @nuxt/ui@4.6.0
npm error Conflicting peer dependency: @inertiajs/vue3@2.3.18
```

--- a/package.json
+++ b/package.json
@@ -214,7 +214,7 @@
     "vue-tsc": "^3.2.6"
   },
   "peerDependencies": {
-    "@inertiajs/vue3": "^2.0.7",
+    "@inertiajs/vue3": "^2.0.7 || ^3.0.0",
     "@nuxt/content": "^3.0.0",
     "joi": "^18.0.0",
     "superstruct": "^2.0.0",

